README.md

```diff
@@ -1,8 +1,13 @@
-# api3-dao-dashboard
+# api3-dao-dashboard [![ContinuousBuild](https://github.com/api3dao/api3-dao-dashboard/actions/workflows/main.yml/badge.svg?branch=main)](https://github.com/api3dao/api3-dao-dashboard/actions/workflows/main.yml)
```

Comment:
I'd prefer the badge under the title rather than in it.

Reply:
My intention was to save vertical space. I can revert though if you still prefer it separate.

Reply:
I haven't seen many repos with it like this, but ok, I don't feel that strongly.